### PR TITLE
Add `votePct` function that allows to partially support/reject votes

### PR DESCRIPTION
**Pull request by sembrestels**
This pull request still needs to be reviewed, please do not merge it yet.

### CODE REVIEW

File: contracts/Voting.sol [hunk 1]
```diff
@@ -440,34 +453,39 @@ contract Voting is IForwarder, AragonApp, BasicMetaTransaction {
...
     /**
     * @dev Internal function to cast a vote. It assumes the queried vote exists.
     */
-    function _vote(uint256 _voteId, bool _supports, address _voter, bool _executesIfDecided) internal {
+    function _vote(uint256 _voteId, uint256 _pct, address _voter, bool _executesIfDecided) internal {
         Vote storage vote_ = votes[_voteId];
 
         VoterState state = vote_.voters[_voter];
         require(state == VoterState.Absent, "Can't change votes");
+        require(_pct != PCT_BASE / 2, "VOTING_CANNOT_ABSTAIN");
```
Comment (kanaagawa) [inline contracts/Voting.sol:468]: What do you think of following the convention of having the error string as a `private constant`?

Comment (kanaagawa) [reply contracts/Voting.sol:468]: Maybe we can also include a comment with the explanation of why this require prevents voting 50%.

**Review by 0xGabi — COMMENTED** [review]
Well done! I highlight how small the changing surface was 👏

File: contracts/Voting.sol (hunk 1 above)
Comment (sembrestels) [reply contracts/Voting.sol:468]: I'm worried to not add extra variables, but I understand constants do not count as them right? My concern is in relation to affect the UnstructuredStorage pattern.

**Pull request closed without merging**